core-services/prow/02_config/_plugins.yaml

```diff
@@ -1793,27 +1793,35 @@ config_updater:
         - bastion-ppc64le-libvirt
         build02:
         - bastion-ppc64le-libvirt
+        build03:
+        - bastion-ppc64le-libvirt
```

Comment:
I would leave the bastion stuff for another PR.
We have lots of other objects than CMs.

Reply:
OK

Reply:
But having the CMs populated would not hurt /shrug

Reply:
Addressed